misaligned.js

```diff
@@ -1,16 +1,17 @@
-const {expect} = require('chai')
+const { expect } = require("chai");
 
 function print_color_map() {
-    const majorColors = ["White", "Red", "Black", "Yellow", "Violet"];
-    const minorColors = ["Blue", "Orange", "Green", "Brown", "Slate"];
-    for (let i = 0; i < majorColors.length; i++) {
-        for (let j = 0; j < minorColors.length; j++) {
-            console.log(`${i * 5 + j} | ${majorColors[i]} | ${minorColors[j]}`);
-        }
+  const majorColors = ["White", "Red", "Black", "Yellow", "Violet"];
+  const minorColors = ["Blue", "Orange", "Green", "Brown", "Slate"];
+  for (let i = 0; i < majorColors.length; i++) {
+    for (let j = 0; j < minorColors.length; j++) {
+      console.log(`${i * 5 + j} | ${majorColors[i]} | ${minorColors[j]}`);
     }
-    return majorColors.length * minorColors.length;
+  }
+  return majorColors.length * minorColors.length;
 }
 
 result = print_color_map();
 expect(result).equals(25);
-console.log('All is well (maybe!)');
+expect(result).equals(24);

```

Comment:
How do we effectively check the correctness of the color map content?
Please think over test cases that can check the pair number and it color map.